bin/deploy-bamboo.sh

```
@@ -89,6 +90,7 @@ dockerRun() {
         -e "NODE_OPTIONS=--max_old_space_size=4096" \
         -e "SITE_BUCKET=${bamboo_SITE_BUCKET}" \
         -e "STAGE_NAME=$bamboo_STAGE_NAME" \
+        -e "STAGING_API_KEY=$bamboo_STAGING_API_KEY" \
```

Shellcheck (0.11.0)

[warning] bamboo_STAGING_API_KEY is referenced but not assigned.

(SC2154)